src/pages/experience-matching/experience-matching-page.tsx

```
@@ -1,7 +1,72 @@
+      <Funnel>
+        <Step name="기업 선택">
+          <div>기업 선택 창입니다</div>
+          <button onClick={() => nextStep()}>선택하기</button>
+        </Step>
+        <Step name="기업 정보 확인">
+          <div>기업 정보 확인 란입니다</div>
+          <button onClick={() => prevStep()}>이전단계</button>
+          <button onClick={() => nextStep()}>다음단계</button>
+        </Step>
+        <Step name="나의 경험 확인">
+          <div>나의 경험 확인 란입니다</div>
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)